bin/plugin/commands/changelog.js

```diff
@@ -683,7 +681,10 @@ async function fetchAllPullRequests( octokit, settings ) {
 function getChangelog( pullRequests ) {
 	let changelog = '## Changelog\n\n';
 
-	const groupedPullRequests = groupBy( pullRequests, getIssueType );
+	const groupedPullRequests = groupBy(
+		skipUsers( pullRequests ),
```

Comment:
Note @priethor we are now skipping "bots" from the changelog output itself. Do we want to do that?

Reply:
Yes, that makes a lot of sense because we have now dependabot polluting the changelog 😅

Reply:
Yes, we should do that 👍